### ISSUE TITLE
Menu should be marked up as an unordered list

### ISSUE THREAD

**Issue by dougmacknz** [cultureamp/kaizen-design-system#1921]
TLDR: Menu should be marked up as a `<ul>` to signal that the items are grouped.

## Audit info

Type: WCAG 2
Principle: Perceivable
Guideline: 1.3 Adaptable
Success Criteria: 1.3.1 Info and Relationships (A)
Level: A
Platform: Web
Severity: Medium

### Why it matters
The menu items are announced by screen readers as unrelated items. Grouping related items, such as the navigation links and buttons, using list elements helps screen reader users to understand the relationship between items and navigate between groups more easily.
When lists are marked up correctly, screen reader users are notified when they come to a list and are told how many items are in the list. Screen reader users can also navigate from list-to-list or item-to-item. This isn’t possible when lists aren’t marked up correctly.

### Our recommendation
Mark up related navigation items (including sub-menus) as unordered lists.

**Comment by lloydstubber** [comment]
Unfortunately, this component needs a bit of an overhaul to pass this audit. As wrapping the `<MenuContent>` in a `ul` the children links in `li` doesn't work.

This is because we don't currently group our links together. So we end up with a `div` and `hr` inside of the `ul`

<img width="887" alt="Screen Shot 2021-09-06 at 12 08 43 pm" src="https://user-images.githubusercontent.com/18339250/132150673-c69ac8eb-4e8f-4998-8976-5fa6ea4c70a0.png">

![image](https://user-images.githubusercontent.com/18339250/132150820-cade8a78-9d6e-4fd8-ad48-044710c3ef81.png)

### The solution?
After speaking with @dougmacknz our best bet is to create a new version of this (to save the old from a breaking change) with a better designed API. Where the lists are split into groups and can have headings. This seems to be what other large companies are doing (e.g. Facebook and Airbnb).
